### ISSUE TITLE
Running from the editor should warn about going over maximum resources

### ISSUE THREAD

**Issue by subsoap** [defold/defold#5173]
Currently when you run a project from the editor it does not appear to warn about going over the maximum number of resources. So when bundling loading errors can appear in a project that don't make sense immediately, and only become evident after bundling a debug build.

Builds ran from the editor should at least warn in the console that the project is currently going over the maximum number of resources set in game.project

Also when toggling the profiler in engine that display should also show the current total resources out of max set resources, that would be helpful.

**Comment by britzl** [comment]
There should be no difference between running from the editor or bundle when it comes to max resource counts etc Are you sure this is the case?

**Comment by subsoap** [comment]
@britzl Yes, 100%. We had max resources in a project set to 1024. Running in editor never complained. Then when bundling release changing screens suddenly broke as we reached above 1024 total resources. Changing to 2048 fixed the issue for us. We had to make a debug bundle to understand what was wrong since only then the problem was being printed, and since it's not reported to the error callback it wasn't immediately obvious like other errors generally are.
